RadixWallet/Features/AssetsFeature/Components/HelperViews/ResourceBalance/ResourceBalanceView.swift

```diff
@@ -465,7 +465,7 @@ extension ResourceBalanceView {
 		}
 
 		private var useSpacer: Bool {
-			amount != nil || fallback != nil
+			amount != nil || fallback != nil || caption1 == nil
```

Comment:
Spacer required for the case of missing caption1 and the checkmark selection needs to be disaplyed.